File: C programs/Fibonacci.c
// Program to calculate a fibonacci sequence

#include <stdio.h>

int main( void ) {
    int number;
    
    // Set intial terms
    int term1 = 0, term2 = 1;
    int nextTerm = ( term1 + term2 );

    // User enters length of sequence
    printf("Please enter an integer: ");
    scanf(" %d", &number);

    // Print fibonacci sequence
    printf("Fibonacci sequence = %d %d ", term1, term2);
    while ( nextTerm <= number ) {
        printf("%d ", nextTerm);

        // Shift numbers
        term1 = term2;
        term2 = nextTerm;
        nextTerm = ( term1 + term2 );
    }
    printf("\n");

    // Exit program
    return 0;
}

File: Python programs/Control Flow/fibonacci.py
# Program to calculate fibonacci series

number = int( input('Enter an integer: ') )

t1 = 1
t2 = 1

print( t1 )
for index in range( 1, number ):
    print( t1 )
    
    result = t1 + t2
    
    t2 = t1
    t1 = result